oio_rest/tests/test_validate.py

```diff
@@ -82,13 +82,12 @@ def setUp(self):
 
     def _json_to_dict(self, filename):
         """
-        Load a JSON file from /interface_test/test_data and return it as JSON
+        Load a JSON file from tests/fixtures and return it as JSON
```

Comment:
```suggestion
        Load a JSON file from ``tests/fixtures`` and return it as JSON
```

…also, not your fault, but directives like `:param:` require an empty line separating them from the preceding paragraph.